### ISSUE TITLE
feature request: skip stacks that are already processed

### ISSUE THREAD

**Issue by JuhaHuiskonen** [alisterburt/autoalign_dynamo#7]
A feature request: it would be nice to have an option to skip the stacks that have already been processed

**Comment by alisterburt** [comment]
Agreed that it could be nice! I'm not using these scripts myself anymore but would happily accept a PR
